README.md

```diff
@@ -759,6 +759,29 @@ If you haven't installed node into your runner image, add the following command
 > [!IMPORTANT]
 > **Be sure to set `NODE_ENV=production`**, so that the SSR script is cached in memory. Otherwise, your page rendering times will be very slow!
 
+> [!NOTE]
+This package supports generating JavaScript files using the **ECMAScript Module (ESM) format**. This is useful if your codebase or environment relies on native ESM imports rather than CommonJS.
+
+```elixir
+{Inertia.SSR,
+  path: Path.join([Application.app_dir(:my_app), "priv"]),
+  esm: true,
+  esm_cache_busting: Mix.env() != :prod,
+  esm_module_extension: "mjs" # No need to be set. This is the default value.
+}
+```
+`:esm_cache_busting` Useful for forcing reloads of your ssr.js file in environments that use dynamic imports. Defaults to false. In most cases, you’ll want to keep this set to false in production, since the generated JS file doesn't change frequently
+
+`:esm_module_extension` Sets the file extension for the generated ESM module. Defaults to "mjs". You’ll need to configure your JS bundler to output the ssr.js file with the .mjs extension so that Node.js recognizes it as an ESM module.
```

Comment:
I added this option but I'm not sure it's the right call. Allowing to pass ssr file extension is helpful because elixir-nodejs under the hood use CommonJS and if you're using esm by setting `esm: true` the file with `.js` will be interpreted as `commonjs` instead of `esm`. But if you're file has the `.mjs` extension you don't need to do nothing 

ESM and the running working on `commonjs` can coexist. 

I really think NodeJS module system is super confusing 😂 but it's what it is and we should push for using ESM whenever we can. CommonJS has to die bit by bit.